Controllers/BranchesController.cs

```diff
@@ -2,51 +2,101 @@
 using System.Linq;
 using System.Threading.Tasks;
 using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
+using StoresManagement.Constants;
 using StoresManagement.Data;
 using StoresManagement.Models;
 using StoresManagement.ViewModels;
 
 namespace StoresManagement.Controllers
 {
+    [Authorize(Roles = "Manager,Administrator,Seller")]
```

Comment:
As you provided constants class and here can be some problem to use it, maybe it is right time to try implement policy based roles authorization https://docs.microsoft.com/pl-pl/aspnet/core/security/authorization/policies?view=aspnetcore-3.1

Reply:
To be studied and considered.

Reply:
Implemented. Code is cleaner now. Thanks. 👍